devel/playdate-sdk/Portfile

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                playdate-sdk
+version             1.12.2
```

Comment:
while this has been in the PR queue it appears a newer version is available, perhaps update to that one?